src/app/containers/Bulletin/index.test.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import BulletinContainer from '.';
+
+const tvBulletinItem = {
+  name: 'Test TV Bulletin promo',
+  summary: 'Test TV summary',
+  indexImage: {
+    id: '63711781',
+    subType: 'index',
+    href: 'http://b.files.bbci.co.uk/4917/test/_63711781_clinton.jpg',
+    path: '/cpsdevpb/4917/test/_63711781_clinton.jpg',
+    height: 371,
+    width: 660,
+    altText: 'Clinton',
+    caption: 'Clinton',
+    copyrightHolder: 'BBC',
+  },
```

Comment:
Looks good 👍 . I'm wondering if the test data should only include items that are being used? For example with `tvBulletinItem.indexImage` only the `href` and `altText` are used

Reply:
Done